Dockerfile

```diff
@@ -87,13 +84,15 @@ RUN cd ${TEMP} \
 \
 # Add group and user
 && groupadd -g ${GID} nexus \
-&& adduser -u ${UID} -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false nexus \
+&& adduser -u ${UID} -M -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false nexus \
+&& cp /etc/passwd /home/jboss/passwd \
```

Comment:
nss_wrapper is configured to find passwd in an alternate location, but adduser added it in the normal place, so copy it. <- this should be a comment in the file i now realize.

Reply:
Can you explain this nss_wrapper a bit more? Googling says this is relating to test functions, but this is our main docker image

Reply:
i think some older versions of openshift suggest nss_wrapper to allow your app to create and use your own user ids (like our nexus user). UBI8 images are geared toward use on openshift, and that's the image we use for both these images. The latest openjdk-8:1.10 sets it up to this alternate passwd location automatically, and chown uses it, but adduser didn't, so I just had to work around it by copying the file we setup. 

Ultimately, we may eventually want to stop using our own UID in kubernetes, but that's a bigger task for another day.

Reply:
as an alternative, we could strip back the base image to ubi-minimal and install red hat's openjdk like the openjdk-8 image would, just without all the extra stuff: https://github.com/sonatype/docker-nexus-iq-server/pull/82